net-im/telegram-desktop-bin/telegram-desktop-bin-3.0.1-r1.ebuild

```diff
@@ -33,9 +33,11 @@ RDEPEND="
 S="${WORKDIR}/Telegram"
 
 src_install() {
-	exeinto /usr/lib/${PN}
-	doexe "Telegram"
-	newbin "${FILESDIR}"/${PN} "telegram-desktop"
+	newbin "Telegram" "telegram-desktop"
+
+	insinto /etc/tdesktop/
```

Comment:
These quotes and trailing / aren't needed, but up to you (it's just style).

Reply:
removed quotes and /